client_lib/pulp/client/commands/repo/importer_config.py

```diff
@@ -98,10 +98,10 @@ def __init__(self):
             parse_func=parsers.pulp_parse_optional_nonnegative_int
         )
 
-        d = _('content downloading policy ({m})'.format(
-            m=' | '.join((constants.DOWNLOAD_IMMEDIATE,
-                          constants.DOWNLOAD_BACKGROUND,
-                          constants.DOWNLOAD_ON_DEMAND))))
+        methods = ' | '.join((constants.DOWNLOAD_IMMEDIATE,
+                              constants.DOWNLOAD_BACKGROUND,
+                              constants.DOWNLOAD_ON_DEMAND))
+        d = _('content downloading policy (%(method)s)' % {'method': methods})
```

Comment:
pretty sure I've seen str.format() used in lots of other parts of sever code.  why an issue here?